components/brave_wallet_ui/components/desktop/network-filter-selector/index.tsx

```diff
@@ -95,7 +95,8 @@ function NetworkFilterSelector () {
                 <SubDropDown>
                   {sortedNetworks.filter((n) =>
                     n.coin === network.coin &&
-                    n.symbol.toLowerCase() === network.symbol.toLowerCase())
+                    n.symbol.toLowerCase() === network.symbol.toLowerCase() &&
+                    n.chainId !== BraveWallet.OPTIMISM_MAINNET_CHAIN_ID)
```

Comment:
maybe add a comment explaining why we hid this here?

Reply:
Added a comment!